languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/multiwords.txt

```diff
@@ -1,4 +1,52 @@
 #French multiwords file used for chunking
+au mieux A
+charge acquéreur N f s
+bas de contention N m sp
+satisfaction client N f s
+relation client N f s
+fiche client N f s
+fiches client N f p
+base de données client N f s
+bases de données client N f p
+un tantinet A
+latina J f sp
+langue maternelle N f s
+mauvais goût N s p
+tout de même A
+Troie Z f s
+a première vue A
+à plusieurs reprises A
+donnée personnelle N f s
+données personnelles N f p
+plan cul N m s
+plans cul N m p
+mis à part P
+tout à coup A
+moyenne générale N f s
+moyennes générales N f p
+forte puissance J f s
+garantie constructeur N f s
+grand public A
+grand champs N m s
+coupe-bise N m s
+coupes bise N m s
+coupe bise N m s
+folk J e sp
+tarif entreprise N m s
+tarifs entreprise N m p
+plus démunis N m p
+par la suite A
+bande dessinée N f s
+bandes dessinées N f p
+double corps N m s
+court A
```

Comment:
Why do we have that?

Reply:
This should not be here. "Court" is obviously an ambiguous word (adjective, verb, adverb), and it should be disambiguated (if possible) in disambiguation.xml.